civihr_employee_portal/views/views_export/views_emergency_contacts.inc

```diff
@@ -85,6 +85,22 @@ $handler->display->display_options['fields']['dependant_s__92']['field'] = 'depe
 $handler->display->display_options['fields']['dependant_s__92']['label'] = '';
 $handler->display->display_options['fields']['dependant_s__92']['exclude'] = TRUE;
 $handler->display->display_options['fields']['dependant_s__92']['element_label_colon'] = FALSE;
+/* Field: Global: Custom text */
```

Comment:
Perhaps we should add some comment to describe this field more and that it is used to place a delete button on the Emergency Contacts and Dependents View also because most of the field values are not descriptive.
I didn't get the true picture until I saw the GIF on the other PR in the description.

Reply:
For auto-exported stuff I don't think adding comments is really a good option because it will probably be overwritten by the next person to do an export. I'll check if we can add some description field to the export that will be imported (and included in the export) or maybe just make the field ID clearer about what it does.

Reply:
Either of the suggested options is fine. A description field seems better though.